gulp build fails due to a missing library file

Following the steps outlined in [DEVELOP.md](https://github.com/hammerlab/cycledash/blob/master/DEVELOP.md), I get the following error when trying to build the project with gulp:

```bash
$ gulp build
[18:44:48] Using gulpfile ~/Projects/cycledash/gulpfile.js
[18:44:48] Starting 'build'...

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: Cannot find module '../lib/querylanguage.js' from '/Users/aksoyb/Projects/cycledash/cycledash/static/js'
    at /Users/aksoyb/Projects/cycledash/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:55:21
    at load (/Users/aksoyb/Projects/cycledash/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/Users/aksoyb/Projects/cycledash/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /Users/aksoyb/Projects/cycledash/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at Object.oncomplete (fs.js:107:15)
```

Here is the problematic line in `QueryLanguage.js`: https://github.com/hammerlab/cycledash/blob/master/cycledash/static/js/QueryLanguage.js#L3

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -124,7 +124,7 @@ gulp.task('staticlibs', function() {
 
 // Default task which compiles the JS and then watches the JS and CSS for
 // changes.
-gulp.task('default', ['watch', 'js', 'sass']);
+gulp.task('default', ['watch', 'js', 'sass', 'peg']);
 
 // Build production resources and copy them into the serving directory.
 gulp.task('prod', ['peg', 'build', 'staticlibs', 'sass']);

